manifests/resource/server.pp

```diff
@@ -24,7 +24,8 @@
 #   [*autoindex*]                  - Set it on 'on' or 'off 'to activate/deactivate autoindex directory listing. Undef by default.
 #   [*proxy*]                      - Proxy server(s) for the root location to connect to.  Accepts a single value, can be used in
 #     conjunction with nginx::resource::upstream
-#   [*proxy_read_timeout*]         - Override the default the proxy read timeout value of 90 seconds
+#   [*proxy_read_timeout*]         - Override the default proxy read timeout value of 90 seconds
```

Comment:
the message is now changed to "Override the default proxy read timeout value of 90 seconds"